### DISCUSSION TITLE
Finding maintained repositories

### DISCUSSION THREAD

**Discussion by pavelskipenes** [community/community#16386] [Other Feature Feedback, Questions, & Ideas]
Assume you found a repository A that is "not being actively maintained" and you are looking for a fork B that is in a more maintained state. Right now the way of finding that fork B is not intuitive.

Current situation:
1. Go to repository A
2. Click on forks
3. Click on every fork and decide manually if the fork is maintained or not

Preferred / Suggestion:
1. Go to repository A
2. Click on forks
3. Forks are sorted with "most" maintained first

One could probably calculate maintenance level by a weighted sum of number of commits after diverge, number of contributors, number of forks and so on. What do you think?

**Comment by kapcod** [comment]
To add another suggestion to complement it, each fork can be given a comment or description to answer "why I did this fork" or "why you should use my fork instead of origin". For example it could be 
1. Fix this bug
2. Add my own features
3. Maintain the abandoned project

In the first case you might want to cherry-pick or fork it, In the second case you wouldn't use it, but in the third you'd just might to switch to it instead of origin.

**Comment by pavelskipenes** [comment]
https://github.com/community/community/discussions/40968
